README.md

```diff
@@ -73,7 +73,7 @@ You can use the FHIR MCP Server by installing our Python package, or by cloning
 
 1. **Configure Environment Variables:**
 
-    To run the server, you must set `FHIR_SERVER_BASE_URL`. If you plan to use authorization, you'll also need to configure `FHIR_SERVER_CLIENT_ID`, `FHIR_SERVER_CLIENT_SECRET`, and `FHIR_SERVER_SCOPES`. By default, the MCP server will listen on http://localhost:8000. You can customize the host and port by setting `FHIR_MCP_HOST` and `FHIR_MCP_PORT` respectively.
+    To run the server, you must set `FHIR_SERVER_BASE_URL`. FHIR server authorization is enabled by default. To enable authorization, you'll also need to configure `FHIR_SERVER_CLIENT_ID`, `FHIR_SERVER_CLIENT_SECRET`, and `FHIR_SERVER_SCOPES`. If you plan to **disable authorization**, `FHIR_SERVER_DISABLE_AUTHORIZATION` to `True`. By default, the MCP server will listen on http://localhost:8000. You can customize the host and port by setting `FHIR_MCP_HOST` and `FHIR_MCP_PORT` respectively.
```

Comment:
Shall we rephrase this section? It seems confusing. May be we can add two points, one to enable and one to disable auth.